### ISSUE TITLE
Selected class not being applied

### ISSUE THREAD

**Issue by CookieMichal** [kylestetz/CLNDR#291]
#### Context
Selected class isn't correctly applied, when changing month by clicking on a day of the next month. It automaticaly fixes itself when changing month back and forth using <kbd><</kbd> and <kbd>></kbd>, or when `.render()` is triggered.

#### Actual Behavior
![behavior](https://vgy.me/gSBOez.gif)

#### Steps to Reproduce
1.  Setup new calendar with
 ```json
trackSelectedDate: true, 
showAdjacentMonths: true,
adjacentDaysChangeMonth: true
```
2. Find a month that doesn't end on Saturday (for 'en' locale)
3. Select day of next month

See https://jsfiddle.net/CookieMichal/0xv07zqz/3/

#### Your Environment
* jQuery 3.1.1
* underscore 1.8.3
* moment.js 2.17.1
* clndr 1.4.7
* Google Chrome 55.0.2883.87

**Comment by mikegioia** [comment]
Thanks @CookieMichal this is an interesting one. It probably has something to do with either the events not triggering / not in the right order, or the render from changing month happening after the selected class is applied.

If you're able to find this one and submit a PR I would appreciate it. Otherwise I'll have to check it out over this or next weekend.

**Comment by mikegioia** [comment]
This is now fixed and deployed in 1.4.9!
